pkg/network/config/config.go

```diff
@@ -223,6 +223,15 @@ func New() *Config {
 		RecordedQueryTypes: cfg.GetStringSlice(join(netNS, "dns_recorded_query_types")),
 	}
 
+	if !cfg.IsSet("max_closed_connections_buffered") && c.MaxClosedConnectionsBuffered < int(c.MaxTrackedConnections) {
```

Comment:
I think for `IsSet` to work, you have to remove the default value.

Reply:
Updated, thanks